osdk/src/config_manager/test/OSDK.toml.full

```diff
@@ -9,7 +8,6 @@ grub-mkrescue = "/usr/bin/grub-mkrescue"
 ovmf = "/usr/bin/ovmf"
 
 [qemu]
-path = "/usr/bin/qemu-system-x86_64"
```

Comment:
The configuration file is named OSDK.toml.full and should contain all possible configurations. I suggest creating `/tmp/initramfs.cpio.gz` and `/tmp/qemu-system-x86_64` to pass the test.